src/allotropy/allotrope/models/light_obscuration_benchling_2023_12_light_obscuration.py

```diff
@@ -130,21 +136,6 @@ class DistributionAggregateDocument:
     distribution_document: list[DistributionDocumentItem]
 
 
-@dataclass
-class Asm:
-    field_asm_manifest: Optional[Union[str, Manifest]] = None
-
-
-@dataclass
-class DeviceControlDocumentItemModel(DeviceControlDocumentItem):
```

Comment:
This change just cleans up the model a bit by combining these two classes that are only used together.